test/test_backbone_utils.py

```diff
@@ -23,6 +24,23 @@ def test_resnet_fpn_backbone(backbone_name):
     y = resnet_fpn_backbone(backbone_name=backbone_name, pretrained=False)(x)
     assert list(y.keys()) == ["0", "1", "2", "3", "pool"]
 
+    with pytest.raises(ValueError):
```

Comment:
As a convention, Please append the error message
```suggestion
    with pytest.raises(ValueError, "Whatever error message it will raise"):
```

Reply:
yes you left that comment on previous PR,

Reply:
This seems to be failing.  Should we add `match=` argument as per https://docs.pytest.org/en/6.2.x/reference.html?highlight=raises#pytest.raises

Here is a simpler test I wrote that fails too with the same error `TypeError: 'Sth' object (type: <class 'str'>) must be callable`:
```
import pytest

def checkme(p):
    raise ValueError("Sth")

@pytest.mark.parametrize("mypar", (1,2,3))
def testMethod(mypar):
    with pytest.raises(ValueError, "Sth"):
        checkme(mypar)
```

Reply:
okay it is happy with the `match=` argument